css_stylesheet.css

```diff
@@ -199,6 +203,45 @@ footer {
 	color: #343A40;
 }
 
+
+@media screen and (max-width: 1024px) and (min-width:770px) {
```

Comment:
Were you having issues with styling from this media query overriding styling from the second one below? If not, then the (min-width: 770px) is redundant.

Reply:
Yes, I wanted the styling to change when changing from iPad to iPhone size screen.

Reply:
It will do that anyway due to the second media query. The portfolioButtons rules from the second media query will override this one, and the rule setting paragraphs to display:block is in both anyway.

Reply:
Ah I get it now. Thank you.